modules/nw-operator-cr.adoc

```diff
@@ -108,12 +104,6 @@ endif::operator[]
 |`object`
 |Configures the network plugin for the cluster network.
 
-|`spec.kubeProxyConfig`
-|`object`
-|
-The fields for this object specify the kube-proxy configuration.
-If you are using the OVN-Kubernetes cluster network plugin, the kube-proxy configuration has no effect.
```

Comment:
FWIW, this is still used by some certified third-party network plugins (notably Calico), but I don't know if we mention those at all in our documentation.

(Admins using ovn-k may also want believe that they want to edit kube-proxy configuration, not realizing that ovn-k doesn't use kube-proxy. So having something about the fact that ovn-k doesn't use kube-proxy may also still be useful.)

Reply:
Put this back in. Good catch, thanks Dan.